### DISCUSSION TITLE
Bing Search not working in hubs cloud

### DISCUSSION THREAD

**Discussion by majorskyworks** [mozilla/hubs#3309] [Hubs Cloud Questions] [answered]
I have been unsuccessful in getting the Bing Search API Key to work inside of Mozilla Hubs cloud. the Sketchfab API key and the Google Poly API key both have worked perfectly, but the search is not. I went into Azure services and created a Cognitive Services key as specified and it just doesn't seem to work. I have tried a Bing Search key, a Cognitive Services key...no success. Hope you can help.

**Comment by robinkwilson** [comment, marked as answer]
https://github.com/mozilla/hubs-cloud/issues/62

We have a bug open to follow!
